test/generate-tests-utils.xspec

```diff
@@ -1,77 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- ===================================================================== -->
-<!--  File:       test/generate-tests-utils.xspec                          -->
-<!--  Author:     Florent Georges                                          -->
-<!--  Tags:                                                                -->
-<!--    Copyright (c) 2010 Jeni Tennsion (see end of file.)                -->
-<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
-<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:test="http://www.jenitennison.com/xslt/unit-test" query="http://www.jenitennison.com/xslt/unit-test" query-at="../src/compiler/generate-query-utils.xql" stylesheet="../src/compiler/generate-tests-utils.xsl">
+<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:test="http://www.jenitennison.com/xslt/unit-test" stylesheet="../src/compiler/generate-tests-utils.xsl">
```

Comment:
I suggest to reinstate the copyright at the beginning and the end of the file. I would personally prefer to have all these copyright infos consolidated in the [license file](https://github.com/xspec/xspec/blob/master/LICENSE) rather than at the beginning of each script but it may be complicated or long winded to get the consent from the original developers. Feel free to add your nickname as reviewer in the author comment or add a comment *extended by*.